### DISCUSSION TITLE
Cant upgrade to latest version of liquidsoap

### DISCUSSION THREAD

**Discussion by fedwa-zarrok** [savonet/liquidsoap#2294] [Installation]
I cant get liquidsoap to update to latest version
Update method : 
**_sudo apt-get update_**
**_sudo apt-get install liquidsoap_** gives me : 

liquidsoap is already the newest version (1.3.3-2).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

**Comment by Vaalyn** [comment]
The recommended way to install Liquidsoap is via the OPAM package manager. Packages provided by the repositories of OS distributions are often times a few version behind on lots of software projects until they release their next OS version.

You can also use the pre-compiled .deb files from the releases here on GH: https://github.com/savonet/liquidsoap/releases/tag/v2.0.3

Download the one matching your OS version and run `dpkg -i` on the .deb file.

Take a look at the install docs of Liquidsoap for more information on how to install LS via OPAM: https://www.liquidsoap.info/doc-dev/install.html

**Comment by fedwa-zarrok** [comment]
Hi @Vaalyn thank you for your response, I already have liquidsoap installed via OPAM and via apt-get command but when I type **_liquidsoap --version_** , I get : 
Liquidsoap 1.3.3
Copyright (c) 2003-2017 Savonet team
Liquidsoap is open-source software, released under GNU General Public License.
See <http://liquidsoap.fm> for more information.

What should I do?

**Comment by TheRealMattLear** [comment]
It sounds like the systems package managed version of liquidsoap is taking precedence, you should remove this ```apt remove liquidsoap``` and reinstall LS following the instructions at https://www.liquidsoap.info/doc-dev/install.html
